src/shared/components/Settings/Preferences/index.jsx

```diff
@@ -88,7 +88,7 @@ export default class Preferences extends React.Component {
       case 'e-mail':
         return <NewsletterPreferencesContainer email={email} />;
       case 'forum':
-        return (window.location.href = `${config.URL.FORUMS}/?module=Settings`) && <LoadingIndicator />;
+        return (window.location.href = `${config.URL.DISCUSSION_FORUM}/profile/preferences`) && <LoadingIndicator />;
```

Comment:
Please use `FORUMS_VANILLA`

Reply:
@luizrrodrigues ok updating PR in sometime